### PR TITLE
Alternative enemy movement animation

### PR DESCRIPTION
**Pull request by CKKH**
- Alternative enemy movement animation branch
- Less animation, cleaner code
- For team consideration

### CODE REVIEW

**Review by hotblack86 — APPROVED** [review]
Great work! Also really good that you've weighed up the pros/cons of having cleaner code vs more features.

File: src/enemy.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def __init__(self, game, x_pos, y_pos):
         pg.sprite.Sprite.__init__(self)
         self.game = game
         self.image = pg.transform.scale(
-            pg.image.load('./images/imp_f0.png'),
+            pg.image.load('./images/skeleton/skeleton_f0.png'),
```
Comment (michaelnguyen974) [inline src/enemy.py:15]: Nice skeleton, we need to give the sprite creators credit/shoutout ;)

**Review by matharotheelf — APPROVED** [review]
Awesome to have multiple enemies

File: src/enemy.py [hunk 2]
```diff
@@ -31,12 +31,16 @@ def move(self):
 
         if movement == ["up"]:
             d_y = -1
+            self.load_up_image()
```
Comment (michaelnguyen974) [inline src/enemy.py:34]: We can try to refactor this, it shall be done

**Review by michaelnguyen974 — APPROVED** [review]

File: src/enemy.py [hunk 3]
```diff
@@ -67,3 +71,31 @@ def update(self):
 
         self.rect.x = self.x_pos * TILESIZE + 1
         self.rect.y = self.y_pos * TILESIZE + 1
+        self.image.set_colorkey(BLACK)
+
+    def load_up_image(self):
```
Comment (michaelnguyen974) [inline src/enemy.py:76]: Nicely abstracted

**Pull request merged**